packages/react-router/rollup.config.mjs

```diff
@@ -1,5 +1,3 @@
-import sourcemaps from 'rollup-plugin-sourcemaps';
```

Comment:
These plugins didn't do anything to the final result, and some of them had peer dependency issues with Rollup 4, so I just removed them.